extensions/extensions-pipeline-invocation-service/src/main/java/com/boozallen/aissemble/pipeline/invocation/service/PipelineInvocationAgent.java

```diff
@@ -101,7 +101,7 @@ protected List<String> buildFinalValuesOverrides(PipelineInvocationRequest reque
         List<String> args = new ArrayList<>();
 
         args.add("--set");
-        args.add("spec.serviceEnabled=false");
+        args.add("service.enabled=false");
```

Comment:
I: this is a miss when migrating the `spec.serviceEnabled` to the `service.enabled` in the `spark-application-chart`